cmd/create/accountroles/cmd.go

```diff
@@ -329,7 +329,7 @@ func run(cmd *cobra.Command, argv []string) {
 		mode, err = interactive.GetOption(interactive.Input{
 			Question: "Role creation mode",
 			Help:     cmd.Flags().Lookup("mode").Usage,
```

Comment:
@ciaranRoche It's always been there, but should we perform the check for `nil` on the `cmd.Flags().Lookup("mode")` . Not a biggie...

Reply:
Yeah lets get a ticket for this, might be worth baking it into milestone 2 of the improvements, we prob should look into a better more deterministic way to approach flags